[TextField] Fix aria-multiline attribute values (#2351)
Shopify/polaris-react: UNRELEASED.md

### Bug fixes

> - Fixed an accessibility issue by updating the `aria-multiline` attribute for `TextField` with `multiline` of type number. Replaced positive number values with `true`. ([#2351](https://github.com/Shopify/polaris-react/pull/2351)).

Reviewer: ```suggestion
- Fixed an accessibility issue with `TextField` `multiline` where `aria-multiline` would be set to an invalid type `number` ([#2351](https://github.com/Shopify/polaris-react/pull/2351))
```
